exercises/word-search/canonical-data.json

```diff
@@ -125,13 +103,8 @@
       "description": "Should locate a left to right word in two line grid",
       "property": "search",
       "input": {
-        "grid": [
-          "jefblpepre",
-          "tclojurerm"
-        ],
-        "wordsToSearchFor": [
-          "clojure"
-        ]
+        "grid": ["jefblpepre", "tclojurerm"],
```

Comment:
although putting them on one line is fine because this case doesn't ask to search for a word vertically, I'm kind of thinking that tracks might want to start presenting each on one line, to start introducing the idea that this should be a grid. And if tracks are going to start doing it, I suggest it's a good idea for the JSON to do it as well

Reply:
Agreed. I've now applied the same multi-line formatting for _all_ array fields. Let me know what that looks like.